lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -65,6 +65,35 @@ public function execute($arguments = [], $options = [])
             QubitTaxonomy::DESCRIPTION_DETAIL_LEVEL_ID => 'detailLevelTypes',
         ]);
 
+        // Define helper function used in post-save logic
+        $relateTermFunction = function ($import, $column, $taxonomyId) {
+            if (!empty($import->rowStatusVars[$column])) {
+                $places = explode('|', $import->rowStatusVars[$column]);
+                for ($i = 0; $i < count($places); ++$i) {
+                    if (empty($places[$i])) {
+                        continue;
+                    }
+
+                    $relation = QubitActor::setTermRelationByName(
+                        $places[$i],
+                        [
+                            'taxonomyId' => $taxonomyId,
+                            'culture' => $import->columnValue('culture'),
+                        ]
+                    );
+
+                    if (null !== $relation) {
```

Comment:
Opinion: For code readability I prefer avoiding nested conditionals when possible.  In this case an alternative to the nesting here is to do:

```
if (null === $relation) {
    continue;
}
```